stubs/inertia/resources/js/Pages/Profile/TwoFactorAuthenticationForm.vue

```diff
@@ -152,7 +152,7 @@
             disableTwoFactorAuthentication() {
                 this.disabling = true
 
-                this.$inertia.delete('/user/two-factor-authentication', {
+                this.$inertia.delete('/user/two-factor-authentication', {}, {
```

Comment:
Inertia now takes an argument body for it's `delete` method, making the signature match the other methods. 
Since we're not posting any data here, I'm just passing in an empty object.

Reply:
@claudiodekker  should this route be updated to use the named route provided by Ziggy?

Reply:
@joelbutcher I looked into this, but because this route is coming from Fortify and isn't named, this isn't possible. Not a bad idea, though!

Reply:
@claudiodekker ah okay, it might be worth a quick PR into Fortify to change this